webapp/components/Map/Map.js

```diff
@@ -31,6 +31,8 @@ L.Marker.prototype.options.icon = L.icon({
 })
 // end of workaround
 
+const initialZoomLevel = 3
```

Comment:
as it is a const of configuration, maybe it is better to put upperCase

```suggestion
const INITIAL_ZOOM_LEVEL = 3
```

Reply:
I had the same doubt .. I agree with you, I'll change it